airqo-platform/AirQo-frontend
# Update device 'status' and 'isActive' fields

#### Summary of Changes (What does this PR do?)
- Adds more device statuses
- Closes #380 

**Note:** This PR depends on the ability to track site position inputs using the siteID on the device edit form

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Checkout the [ft-addDeviceStatusOptions branch](https://github.com/airqo-platform/AirQo-frontend/tree/ft-addDeviceStatusOptions) and follow the instructions in the [README](https://github.com/airqo-platform/AirQo-frontend/blob/ft-addDeviceStatusOptions/netmanager/README.md) to run the application locally.
* Go to the [Device Registry section](http://localhost:5000/registry)
* You should see a variety of statuses on various devices.
* You can also choose to edit and add another deployment status

#### What are the relevant tickets?
- [PLAT-1010](https://airqoteam.atlassian.net/browse/PLAT-1010?atlOrigin=eyJpIjoiNjYzYWNmNzgwNmI1NDI0NjlhNWFhODZjMDJkNGVjZWMiLCJwIjoiaiJ9)
- #380

#### Screenshots (optional)
![image](https://user-images.githubusercontent.com/41787587/156641820-2af2f44b-f6e3-454d-90c6-50831eda309e.png)
